pkg/dochandler/handler_test.go

```diff
@@ -32,6 +31,8 @@ import (
 	"github.com/trustbloc/sidetree-core-go/pkg/patch"
 	"github.com/trustbloc/sidetree-core-go/pkg/processor"
 	"github.com/trustbloc/sidetree-core-go/pkg/versions/0_1/doccomposer"
+	"github.com/trustbloc/sidetree-core-go/pkg/versions/0_1/doctransformer/didtransformer"
+	"github.com/trustbloc/sidetree-core-go/pkg/versions/0_1/doctransformer/doctransformer"
```

Comment:
`doctransformer/doctransformer` is an interesting path. Perhaps the parent folder needs a more generic name?

Reply:
it was based on the same pattern for validator: 
	"github.com/trustbloc/sidetree-core-go/pkg/versions/0_1/docvalidator/didvalidatorr"
	"github.com/trustbloc/sidetree-core-go/pkg/versions/0_1/docvalidator/docvalidator"
The versions folders are prefixed with either doc, operation or txn depending on whether the functionality is related to document, operation or transaction. Of course we can look at all folders to see if we can come up with better names.